kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/utils/SinkRecordConverter.java

```diff
@@ -0,0 +1,47 @@
+package com.wepay.kafka.connect.bigquery.utils;
+
+import com.google.cloud.bigquery.InsertAllRequest;
+import com.wepay.kafka.connect.bigquery.api.KafkaSchemaRecordType;
+import com.wepay.kafka.connect.bigquery.config.BigQuerySinkTaskConfig;
+import com.wepay.kafka.connect.bigquery.convert.KafkaDataBuilder;
+import com.wepay.kafka.connect.bigquery.convert.RecordConverter;
+import org.apache.kafka.connect.sink.SinkRecord;
+
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * A class for converting a {@link SinkRecord SinkRecord} to {@link InsertAllRequest.RowToInsert BigQuery row}
+ */
+
+public class SinkRecordConverter {
+    private BigQuerySinkTaskConfig config;
+    private RecordConverter<Map<String, Object>> recordConverter;
+
+    public SinkRecordConverter(BigQuerySinkTaskConfig config) {
+        this.config = config;
```

Comment:
Would prefer this class takes recordConverter, boolean sanitizeFieldName, (optional) keyName, (optional) dataName. This makes the class cleaner to mock.

Reply:
Will update accordingly.